sdk/program/Cargo.toml

```diff
@@ -37,6 +37,9 @@ blake3 = "0.3.7"
 curve25519-dalek = "2.1.0"
 rand = "0.7.0"
 solana-logger = { path = "../../logger", version = "=1.7.0" }
+hkdf = "0.11.0"
```

Comment:
nit: alphabetical

Reply:
Fixed in 01ed0b9d229749506e0065fed170cac1310741b1